flow/project.py

```diff
@@ -2129,8 +2658,8 @@ def submit(self, bundle_size=1, jobs=None, names=None, num=None, parallel=False,
         :type num:
             int
         :param parallel:
-            Execute all bundled operations in parallel. Does nothing with the
-            default behavior or `bundle_size=1`.
+            Execute all bundled operations in parallel. Will make both bundles and groups run in
+            parallel. If not careful this can lead to large resource requests per job.
```

Comment:
I am not sure to the solution to this (if we even need a solution) since this will only effect people who opt in to using groups, I am fine with this being as it is now (we already don't error on bad resource handling with groups).